### DISCUSSION TITLE
I'd like to have a cloud icon without a download and upload inner icon

### DISCUSSION THREAD

**Discussion by JoshuaBehrens** [shopwareArchive/meteor-icon-kit#26] [Request: Icon] [answered]
I wanted to use a cloud as icon. But you only have a cloud with sub-/inner-icons.

**Comment by Weltraumakustik** [comment, marked as answer]
Thank you for your idea. 
Your suggested icon will be part of the next (#35) release! 🎉 

It will be available in both regular and solid. 

<img width="58" alt="Bildschirm­foto 2023-01-20 um 17 12 19" src="https://user-images.githubusercontent.com/1750320/213748238-3ec4290a-4071-4317-8a65-5ee221d4ff8f.png">
